package.json

```diff
@@ -67,9 +67,8 @@
     "@actions/github": "5.0.3",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 20 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26407&commitId=0a75ec64d4c2605b7acfcb78bb6024bab06b6f08&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1755 screenshots unchanged, 0 screenshots with different dimensions and 20 screenshots with visible difference.


  <details><summary><b>unknown</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CheckboxField converged.horizontal.default.chromium.png|15| Changed|
|CheckboxField converged.validation.default.chromium.png|65| Changed|
|ComboboxField converged.horizontal.default.chromium.png|752| Changed|
|ComboboxField converged.validation.default.chromium.png|65| Changed|
|InputField converged.horizontal.default.chromium.png|752| Changed|
|InputField converged.validation.default.chromium.png|65| Changed|
|ProgressField converged.horizontal.default.chromium.png|752| Changed|
|ProgressField converged.validation.default.chromium.png|65| Changed|
|RadioGroupField converged.horizontal.default.chromium.png|752| Changed|
|RadioGroupField converged.validation.default.chromium.png|65| Changed|
|SelectField converged.horizontal.default.chromium.png|752| Changed|
|SelectField converged.validation.default.chromium.png|65| Changed|
|SliderField converged.horizontal.default.chromium.png|752| Changed|
|SliderField converged.validation.default.chromium.png|65| Changed|
|SpinButtonField converged.horizontal.default.chromium.png|752| Changed|
|SpinButtonField converged.validation.default.chromium.png|65| Changed|
|SwitchField converged.horizontal.default.chromium.png|752| Changed|
|SwitchField converged.validation.default.chromium.png|65| Changed|
|TextareaField converged.horizontal.default.chromium.png|752| Changed|
|TextareaField converged.validation.default.chromium.png|65| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>